lib/perl/Genome/Config/AnalysisProject/Command/Deprecate.pm

```diff
@@ -0,0 +1,55 @@
+package Genome::Config::AnalysisProject::Command::Deprecate;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Config::AnalysisProject::Command::Deprecate {
+    is => 'Command::V2',
+    has_input => [
+       analysis_projects  => {
+            is                  => 'Genome::Config::AnalysisProject',
+            doc                 => 'the analysis projects to deprecate',
+            is_many             => 1,
+            shell_args_position => 1,
+        }
+    ],
+};
+
+sub help_brief {
+    return 'deprecate the analysis project';
+}
+
+sub help_synopsis {
+    return "genome config analysis-project deprecate <analysis-projects>";
+}
+
+sub help_detail {
+    return <<"EOS"
+Given some analysis projects, this will deprecate the analysis project.  This includes setting all config profile items to disabled and updating the project status to Deprecated.
```

Comment:
There is a disagreement in the number of analysis projects in the first sentence.